packages/template/tsconfig.json

```diff
@@ -1,11 +0,0 @@
-{
```

Comment:
Bonus: removed unused file (I added it by accident in https://github.com/blockprotocol/blockprotocol/pull/55)